scripts/test/run-tests.sh

```diff
@@ -60,20 +55,32 @@ if [ "$1" == "--prebuilt-image" ]; then
 
 else
   if [ "$1" == "--all" -o "$1" == "--only" -a "$2" == "qemux86_64" ]; then
+    wget --progress=dot:giga -N ${UBUNTU_IMAGE_URL} -P input/
     convert_and_test "qemux86_64" \
                      "release-1" \
-                     "${UBUNTU_IMAGE_URL}" \
-                     "${UBUNTU_IMAGE}" \
-                     "${UBUNTU_IMAGE}.gz" \
-                     "configs/qemux86-64_config" || test_result=$?
+                     "input/Ubuntu-Bionic-x86-64.img.gz" \
+                     "gz" \
+                     "--config configs/qemux86-64_config" || test_result=$?
+
+    convert_and_test "qemux86_64" \
+                     "release-1" \
+                     "input/Ubuntu-Bionic-x86-64.img" \
+                     "gz" \
+                     "--config configs/qemux86-64_config" || test_result=$?
```

Comment:
Why are there duplicate entries here?

I assume that we are testing with automatic decompress/compress and "old" method, but then I am confused by the arguments, as the second one should be `"none"` ?

Reply:
Error, clearly. Thx